Common/UnitDefinitions/ElectricAdmittance.json

```diff
@@ -1,7 +1,9 @@
 {
   "Name": "ElectricAdmittance",
   "BaseUnit": "Siemens",
-  "XmlDocSummary": "Electric admittance is a measure of how easily a circuit or device will allow a current to flow. It is defined as the inverse of impedance. The SI unit of admittance is the siemens (symbol S).",
+  "XmlDocSummary": "Electric admittance is a measure of how easily a circuit or device will allow a current to flow by the combined effect of conductance and susceptance in a circuit. It is defined as the inverse of impedance. The SI unit of admittance is the siemens (symbol S).",
+  "XmlDocRemarks": "https://en.wikipedia.org/wiki/Electrical_admittance",
+  "ObsoleteText": "Admittance is a complex number, which is not currently supported by UnitsNet. Please use either ElectricConductance or ElectricSusceptance instead.",
```

Comment:
I see ElectricAdmittance is being obsoleted in favor of ElectricConductance and ElectricSusceptance. This is probably reasonable, but I don't know this domain well enough to say if this is a long term or short term solution. 

Is it likely we would add back EletricAdmittance at a later point?
If so, what would supporting imaginary numbers look like in UnitsNet?

Reply:
ElectricAdmittance and ElectricImpedance are two sides of the same coin. From the [Wikipedia page on admittance](https://en.wikipedia.org/wiki/Admittance): "admittance Y, measured in siemens, is defined as the inverse of impedance Z, measured in ohms."
Y≡1/Z

Beyond this is where my understanding of the math and theory gets fuzzy. I'm going to stick with impedance, because the math and units for that are directly usable in modelling radio antennas and predicting their performance. Impedance is a complex number, made up of a real part (the resistance, R, or the opposition to the flow of any current), and an [imaginary part](https://en.wikipedia.org/wiki/Imaginary_unit) (the reactance, X, or the opposition to the flow of alternating current), thus:
Z=R+jX, where j<sup>2</sup> = -1

(As an aside, why j and not i? In contexts in which use of the letter i is ambiguous or problematic, the letter j is sometimes used instead. For example, in electrical engineering and control systems engineering, the imaginary unit is normally denoted by j instead of i, because i is commonly used to denote electric current.)

So, while resistance is the only real number, and even though reactance is an imaginary number, it isn't too much of a mental leap to use an ElectricReactance class right now that stores its value in a double and just leaves out the imaginary symbol (j). The conversion of an ElectricReactance number back to an imaginary number, involving prepending a 'j' onto a ToString() call, is currently "left as an exercise for the reader".

Supporting complex numbers would either involve using the System.Numerics.Complex implementation in .NET, which would introduce a dependency on System.Numerics, or developing a UnitsNet implementation that would look like the reference source at https://github.com/microsoft/referencesource/blob/master/System.Numerics/System/Numerics/Complex.cs

In the particular case of ElectricImpedance, it would have to be a container class made up of one instance each of two other classes, an ElectricResistance class and an ElectricReactance class. For example, if the ElectricResistance class had a value of 31.8 Ω and the ElectricReactance class had a value of 2.92 Ω (or perhaps "j 2.92 Ω" at some point?), then ElectricImpedance.ToString() would create "31.8 + j 2.92 Ω".

This is why I obsoleted the existing ElectricAdmittance, and I "pre-obsoleted" the ElectricImpedance class as a kind of bookmark that says "Yes, we know this is a unit, but we don't have a technical solution at the moment." At this point, we are getting close to my suggestion of "use a named Tuple and do it yourself".

Reply:
Thanks for a detailed explanation, makes sense to me.

Reply:
That's great- I think it may be possible in the future to come up with a solution which models this correctly (perhaps something like IComplexQuantity<TRealUnit, TImaginaryUnit, TValue> which could itself extend some `ICompositeQuantity<TFirstUnit, TSecondUnit, TValue>` or something of the sort..). 

Fun exercise for the _relatively far future..._

In the meantime, do you think the complex quantities have _any_ practical use, other than as a "documentation-section"? If not, I propose that we stash these two in PR and pin it as "Draft" or something.. It's kinda bugging me to have these "Is now obsolete.." warnings in the build output..